python/tvm/driver/tvmc/compiler.py

```diff
@@ -153,7 +153,10 @@ def drive_compile(args):
         Zero if successfully completed
 
     """
-    tvmc_model = frontends.load_model(args.FILE, args.model_format, args.input_shapes)
+    try:
+        tvmc_model = frontends.load_model(args.FILE, args.model_format, args.input_shapes)
+    except FileNotFoundError:
+        raise TVMCException(f"Input file {args.FILE} does not exist!")
```

Comment:
what about just checking if `os.exists(args.FILE)` rather than assuming all FileNotFoundError relate to the input file?

Reply:
@areusch yeah I was unsure about that option or the one I've used. Sure, let's use `os.path`. I just used `os.path.isfile()` instead of `os.path.exists()` because `exists()` will also return true if `FILE` is directory, which is also invalid in that case. PTAL. Thanks!